### ISSUE TITLE
Support radial-gradient

### ISSUE THREAD

**Issue by outoftime** [fidian/PrettyCSS#38]
Hello,

I noticed that `radial-gradient` and the associated vendor properties are not recognized, though `linear-gradient` et. al. are. Seems like an oversight?

Thanks for the great library!
Mat

**Comment by fidian** [comment]
I would rather add proper support for the properties.  Would you be willing to assemble links to the CSS specs for the various browsers?  For instance, typically I have been using the [CSS3 spec](https://www.w3.org/TR/css3-images/#radial-gradients), useful [Webkit Blog Posts](https://webkit.org/blog/175/introducing-css-gradients/) and [Firefox's documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/radial-gradient).  Can you help me find references for the other browsers or variations on the syntax?

With the rules as found on other websites, I am able to make a parser that properly validates the value instead of just ignoring it.

**Comment by outoftime** [comment]
@fidian okay sure, I’ll see what I can come up with. thanks!

**Comment by outoftime** [comment]
@fidian here you go:

* [CSS3 Spec](https://www.w3.org/TR/css3-images/#radial-gradients) (you have this one)
* [February 2011 CSS3 Draft Spec](https://www.w3.org/TR/2011/WD-css3-images-20110217/#radial-gradients)
* [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/radial-gradient) (you have this one too)
* [WebKit blog post](https://webkit.org/blog/1424/css3-gradients/) (this is more recent and covers `-webkit-radial-gradient`, which I believe has the same syntax as the unprefixed version)
* [MSDN blog post](https://blogs.msdn.microsoft.com/ie/2012/06/25/unprefixed-css3-gradients-in-ie10/) (has examples of older `-ms-radial-gradient` although the article is about moving to the unprefixed versions)
* [Opera blog post](https://dev.opera.com/articles/css3-radial-gradients/) (about gradients in general, but includes examples of `-o-radial-gradient`)

TL;DR: As far as I can see, the `-moz`, `-webkit`, and `-o` vendor prefixed properties use the same syntax as the current unprefixed spec. It appears from the blog post that the `-ms` vendor prefix used the older draft spec (so e.g. `cover` is an allowed size). And of course there is `-webkit-gradient`.

Let me know if that’s helpful—I must admit I am not a CSS expert, so there might be subtlety I’m missing.
